src/utils/fetchClient.ts

```diff
@@ -33,7 +43,16 @@
~
~export const client = {
~  get: <T>(url: string) => request<T>(url),
```

Comment:
Update 'removeComment' function to return an updated list of comments after deleting a comment. Add return type 'Promise<Comment[]>' and add .then() after client.delete to fetch the updated list.